pkg/schema/annotations.go

```diff
@@ -512,6 +574,20 @@ func processOptionalAnnotation(node yamlmeta.Node, optionalAnnotation template.A
 				return nil, err
 			}
 			return defaultAnn, nil
+		case AnnotationDeprecated:
+			if _, ok := node.(*yamlmeta.Document); ok {
+				return nil, schemaAssertionError{
+					description:  fmt.Sprintf("@%v not supported on a document", AnnotationDeprecated),
```

Comment:
To be consistant with other errors (see pkg/schema/annotations.go:559)
```suggestion
					description:  fmt.Sprintf("Invalid schema - @%v not supported on a %s", AnnotationDeprecated, yamlmeta.TypeName(node)),
```

Reply:
Thanks for the comment! I had removed "Invalid schema" to avoid duplicate error message. Made remaining changes.